### DISCUSSION TITLE
👋 Who is using Apollo Angular?

### DISCUSSION THREAD

**Discussion by kamilkisiela** [the-guild-org/apollo-angular#1513] [General]
We would love to hear who's using Apollo Angular and how you're using it :)

If it's okay to share, please mention the name of your company or product so we could include it on a special section on the website.

**Comment by TomasMoskala** [comment]


**Reply by kamilkisiela** [reply]
I don't think this discussion is the right place where your comment should be posted in. If you have any issues, simply submit an issue or contact me directly.

**Comment by TomasMoskala** [comment]


**Reply by kamilkisiela** [reply]
Please don't spam this discussion, I told you what do to. If I didn't respond then I probably had no time to look at it

**Comment by KeithGillette** [comment]
> We would love to hear who's using Apollo Angular and how you're using it :)
> 
> If it's okay to share, please mention the name of your company or product so we could include it on a special section on the website.

[TaskTrain](https://TaskTrain.app) uses Apollo Angular in our PWA client. We thank you for maintaining this library to let us access our GraphQL API the Angular way!

**Comment by alexbjorlig** [comment]
We at [21RISK](https://21risk.com) are using the Apollo Angular package in our application - thanks 🔥
